docker/parse-html/app.rb

```diff
@@ -124,3 +131,29 @@ def extract_abbreviations(html_doc)
 
   abbreviations
 end
+
+def extract_image(html_doc)
+  images = []
+
+  html_doc.css("img").each do |images|
+    images.push({
+      "title" => images.attribute("title"), # Expansion
+      "text" => images.text, # Images
+    })
+  end
```

Comment:
Happy to pair with you on this. I think something like the suggestion below might work, referring to https://www.w3schools.com/tags/tag_img.as. It would need a test for if the `alt` attribute is missing.

```suggestion
  html_doc.css("img").each do |image|
    images.push({
      "alt" => image.attribute("alt")
    })
  end
```